### PR TITLE
Choose from options

### PR DESCRIPTION
**Pull request by jozw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -460,37 +460,19 @@ Translations of the guide are available in the following languages:
...
+  ```Ruby
+  # bad - need to consult first line to understand second line
+  one.two.three.
+    four
 
-    # good - it's immediately clear that the expression continues beyond the first line
-    one.two.three.
-      four
-    ```
+  # good - it's immediately clear what's going on the second line
```
Comment (rberecka) [inline README.md]: Did you mean going on on ?

**Comment by rberecka** [comment]
One small question, but 👍 !
And 😄

File: README.md (hunk 1 above)
Comment (jozw) [inline README.md]: I'll change is to "happening on"

Comment (jozw) [inline README.md]: actually, "going on" works. we expect a `.` to be going on the second line.

**Pull request merged**